src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.logic.commands.deliverycommand;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ORDER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.commands.results.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.delivery.Delivery;
+import seedu.address.model.deliverymodel.DeliveryModel;
+
+/**
+ * Adds a delivery to the delivery book.
+ */
+public class DeliveryAddCommand extends DeliveryCommand {
+
+    public static final String COMMAND_WORD = "add-d";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a delivery to the delivery book. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_PHONE + "QUANTITY "
+            + PREFIX_ADDRESS + "SUPPLIER "
```

Comment:
Should be address, not supplier

Reply:
And should be phone, not quantity! Hahah

Reply:
HAHA i think we should just commit on his behalf. it's a minor issue

Reply:
My bad, Thanks for pointing it out hahah